pac_windows.go

```diff
@@ -0,0 +1,91 @@
+package ieproxy
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+type WINHTTP_AUTOPROXY_OPTIONS struct {
```

Comment:
Only this one or `WINHTTP_PROXY_INFO` as well?
Should the `WINHTTP_AUTOPROXY_*` constants also have a lowercase letter prefix?